pyproject.toml

```diff
@@ -100,7 +100,9 @@ ignore = [
     # Asserts
     "S101",
     # Function Arguments
-    "PLR0913"
+    "PLR0913",
+    # UTC Timezones
+    "DTZ003"
```

Comment:
This is a genuine error.

Reply:
No it isnt.
This error concerns the usage of datetime.utcnow(). I used this because ruff dislikes having datetime.datetime() without a timezone.
Because we dont use the timezone anyways (we only use timestamps and time difference calculations) we can dismiss this error.

Reply:
Is there a reason not to use `datetime.datetime.now(tz=datetime.UTC)`?

Reply:
Yes. Its longer (there really isnt but its not that important)